scripts/india_census/primary_census_abstract_data/readme.md

```diff
@@ -0,0 +1,67 @@
+# Primary Census Abstract Data Tables (India, States/UTs - District Level) Overview
+
+
+### Download URL
+It's available at [censusindia.gov.in](http://censusindia.gov.in/pca/DDW_PCA0000_2011_Indiastatedist.xlsx). You can pre-download the dataset and place it in the `data` folder, or you can download it from the URL in real-time. 
+
+### Overview
+
+The first nine columns refer to the location for which the data is tabulated. The 9th column is called TRU; it stands for Total, Urban, or Rural. From Column 10-93, we have values tabulated. The column header is the census variable. For example, the 10th column header `No_HH`, which has the values for `Number of Households`. A list of all such variables and their definition can be found in the CSV  `india_census/common/primary_abstract_data_variables.csv`
+
+ - State - Two digit state code
+ - District - Three digit district code inside the specific state
+ - Subdistt - Five digit sub-district code inside the specific state
+ - Town/Village - Six digit town or village code
+ - Ward - Four digit ward code
+ - EB - Six digit Enumeration Block Number
+ - Level - If the actual location is India (country), STATE, DISTRICT, SUB-DISTRICT, TOWN, VILLAGE or EB
+ - Name - Official Name of the place
+ - TRU - Stands for Total, Urban or Rural
+ - 10-93 - Data value columns 
+
+
+#### Raw Data Snippet
+
+| State | District | Subdistt | Town/Village | Ward | EB     | Level | Name            | TRU   | No_HH     | TOT_P      | ... |
+| ----- | -------- | -------- | ------------ | ---- | ------ | ----- | --------------- | ----- | --------- | ---------- | --- |
+| 00    | 000      | 00000    | 000000       | 0000 | 000000 | India | India           | Total | 249501663 | 1210854977 | ... |
+| 00    | 000      | 00000    | 000000       | 0000 | 000000 | India | India           | Rural | 168612897 | 833748852  | ... |
+| 00    | 000      | 00000    | 000000       | 0000 | 000000 | India | India           | Urban | 80888766  | 377106125  | ... |
+| 01    | 000      | 00000    | 000000       | 0000 | 000000 | STATE | JAMMU & KASHMIR | Total | 2119718   | 12541302   | ... |
+| ....  |
+| ....  |
+
+
+#### Cleaned Data
+As part of the cleaning process, we remove the unwanted columns and then convert the columns into rows. Then add the `StatisticalVariable` column corresponding to census variable and TRU.
+
+Cleaned data [IndiaCensus2011_Primary_Abstract_Data.csv](IndiaCensus2011_Primary_Abstract_Data.csv) will have the following columns.
+
+- census_location_id - Census location id
```

Comment:
To confirm is this census_location_id or indianCensusAreaCode{YYYY} ?

Reply:
We are using  indianCensusAreaCode{YYYY}/indianCensusAreaCode2011 I have left the csv column name as it is.

I have updated the documentation.